lib/box/index.js

```diff
@@ -0,0 +1,16 @@
+import angular from 'angular';
+
+var box = angular.module('guBox', []);
+
+box.directive('guBox', function boxDirective(){
+  return {
+    restrict: 'AE',
```

Comment:
These containers are usually going to reside within a `gu-column`. I figure writing:

```html
<gu-column>
  <gu-box variant="primary">
    content
  </gu-box>
</gu-column>
```

is pretty verbose. However:

```html
<gu-column gu-box="primary">Content</gu-column>
```

is better. I also wanted to maximise flexibility so didn't want to restrict it to just an attribute.